frontera/contrib/scrapy/converters.py

```diff
@@ -7,6 +7,8 @@
 from frontera.utils.converters import BaseRequestConverter, BaseResponseConverter
 from w3lib.util import to_bytes, to_native_str
 
+from frontera.utils.misc import dict_to_unicode, dict_to_bytes
+
 
 class RequestConverter(BaseRequestConverter):
```

Comment:
I don't see how we could fix this for all possible Scrapy customizations (middlewares, spiders code). Scrapy perfectly allows user to store keys in values using any type and then assures user that he could rely on that storage, and there will be no changes later. 
I'm afraid the only way to fix that in all possible scenarios is to persist string type during serialization.